google/privacy/dlp/v2beta1/dlp_gapic.yaml

```diff
@@ -0,0 +1,201 @@
+  go:
+    package_name: cloud.google.com/go/privacy/dlp/apiv2beta1
+  csharp:
+    package_name: Google.Privacy.Dlp.V2beta1
+  ruby:
+    package_name: Google::Cloud::Privacy::Dlp::V2beta1
+  php:
+    package_name: Google\Cloud\Privacy\Dlp\V2beta1
+  nodejs:
+    package_name: dlp.v2beta1
+license_header:
+  copyright_file: copyright-google.txt
+  license_file: license-header-apache-2.0.txt
+interfaces:
+- name: google.privacy.dlp.v2beta1.DlpService
+  collections:
+  - name_pattern: inspect/results/{result}
+    entity_name: result
+  retry_codes_def:
+  - name: idempotent
+    retry_codes:
+    - UNAVAILABLE
+    - DEADLINE_EXCEEDED
+  - name: non_idempotent
+    retry_codes: []
+  retry_params_def:
+  - name: default
+    initial_retry_delay_millis: 100
+    retry_delay_multiplier: 1.3
+    max_retry_delay_millis: 60000
+    initial_rpc_timeout_millis: 20000
+    rpc_timeout_multiplier: 1
+    max_rpc_timeout_millis: 20000
+    total_timeout_millis: 600000
+  methods:
+  - name: InspectContent
+    flattening:
+      groups:
+      - parameters:
+        - inspect_config
+        - items
+    required_fields:
+    - inspect_config
+    - items
+    request_object_method: true
+    retry_codes_name: non_idempotent
+    retry_params_name: default
+    timeout_millis: 60000
+  - name: RedactContent
+    flattening:
+      groups:
+      - parameters:
+        - inspect_config
+        - items
+        - replace_configs
+    required_fields:
+    - inspect_config
+    - items
+    - replace_configs
+    request_object_method: true
+    retry_codes_name: non_idempotent
+    retry_params_name: default
+    timeout_millis: 60000
+  - name: CreateInspectOperation
+    flattening:
+      groups:
+      - parameters:
+        - inspect_config
+        - storage_config
+        - output_config
+    required_fields:
+    - inspect_config
+    - storage_config
+    - output_config
+    request_object_method: true
+    retry_codes_name: non_idempotent
+    retry_params_name: default
+    timeout_millis: 60000
+  - name: ListInspectFindings
+    flattening:
+      groups:
+      - parameters:
+        - name
+    required_fields:
+    - name
+    request_object_method: true
+    retry_codes_name: idempotent
+    retry_params_name: default
+    field_name_patterns:
+      name: result
+    timeout_millis: 60000
+  - name: ListInfoTypes
+    flattening:
+      groups:
+      - parameters:
+        - category
+        - language_code
+    required_fields:
+    - category
+    - language_code
+    request_object_method: true
+    retry_codes_name: idempotent
+    retry_params_name: default
+    timeout_millis: 60000
+  - name: ListRootCategories
+    flattening:
+      groups:
+      - parameters:
+        - language_code
+    required_fields:
+    - language_code
+    request_object_method: false
+    retry_codes_name: idempotent
+    retry_params_name: default
+    timeout_millis: 60000
+- name: google.longrunning.Operations
```

Comment:
We don't normally have an interface config for long running operations - see e.g. Speech. Do we need one in this case?